plugins/prometheus/mana.go

```diff
@@ -6,10 +6,16 @@ import (
 )
 
 var (
-	accessManaMap       *prometheus.GaugeVec
-	accessPercentile    prometheus.Gauge
-	consensusManaMap    *prometheus.GaugeVec
-	consensusPercentile prometheus.Gauge
+	accessManaMap             *prometheus.GaugeVec
+	accessPercentile          prometheus.Gauge
+	consensusManaMap          *prometheus.GaugeVec
+	consensusPercentile       prometheus.Gauge
+	averageNeighborsAccess    prometheus.Gauge
+	averageNeighborsConsensus prometheus.Gauge
+	averageAccessPledgeBM1    prometheus.Gauge
+	averageAccessPledgeBM2    prometheus.Gauge
+	averageConsensusPledgeBM1 prometheus.Gauge
+	averageConsensusPledgeBM2 prometheus.Gauge
```

Comment:
these would also have to be `*prometheus.GaugeVec`  (or just make two, and label data with base mana type and nodeID)